src/Kount/Ris/Response.php

```diff
@@ -591,6 +591,24 @@ protected function safeGet ($key) {
   }
 
   /**
+   * Safely get all of the values from the RIS response.
+   *
+   * @return array Response fields
+   */
+  protected function safeGetResponseAsDict () {
+    return !is_null($this->response) ? $this->response : '';
+  }
+
+  /**
+   * Get an array with the fields from the RIS response
+   *
+   * @return array Response fields
+   */
+  public function getResponseAsDict () {
```

Comment:
You can safely combine those two functions, removing the protected one. Also, the comment may state that getting the value is null-safe.